packages/tools/devtools/devtools-core/src/data-visualization/DataEditing.ts

```diff
@@ -5,39 +5,37 @@
 
 import { ISharedObject } from "@fluidframework/shared-object-base";
 import { Serializable } from "@fluidframework/datastore-definitions";
-import { EditType } from "../CommonInterfaces";
-
-// Ideas:
-// - Hold onto previous summary and only transmit diff?
-
-// TODOs:
-// - Dependency tracking
-//   - When a particular DDS is no longer reachable via the input data, we need to remove it from the map and stop
-//     emitting updates.
+import { EditType, FluidObjectId } from "../CommonInterfaces";
 
 /**
- * The type of a shared object.
+ * Generates a description of the edit to be applied to {@link @fluidframework/shared-object-base#ISharedObject}'s
```

Comment:
I don't think this phrasing is exactly right. This isn't generating a description of an edit; it's actually making an edit on the associated shared object. We should probably update this documentation to be a bit more clear.